### PR TITLE
Display email address 

### PR DESCRIPTION
**Pull request by hamza221**
Before:
image
After:
image

Fixes https://github.com/nextcloud/mail/issues/8379

### CODE REVIEW

**Comment by mat-m** [comment]
I would prefer to keep the labels and have the mail address on hovering.

**Comment by ChristophWurst** [comment]
@hamza221 please create a ticket for this change so that we can discuss the visual changes with designers.

This change originates from https://github.com/nextcloud/mail/issues/7884.

File: src/components/ThreadEnvelope.vue [hunk 1]
```diff
@@ -60,6 +61,9 @@
 				@click.native.prevent="$emit('toggle-expand', $event)">
 				<div class="sender">
 					{{ envelope.from && envelope.from[0] ? envelope.from[0].label : '' }}
+					<p class="sender-email">
```
Comment (ChristophWurst) [inline src/components/ThreadEnvelope.vue:64]:
```suggestion
					<p class="envelope__header__sender-email">
```

for the BEM convention

File: src/components/ThreadEnvelope.vue [hunk 2]
```diff
@@ -652,6 +660,7 @@ export default {
 			padding: 10px;
 			border-radius: var(--border-radius);
 			min-height: 68px; /* prevents jumping between open/collapsed */
+
```
Comment (ChristophWurst) [inline src/components/ThreadEnvelope.vue:663]: unrelated :)

**Review by ChristophWurst — COMMENTED** [review]
Code looks good

Please update the screenshots in the PR description

**Comment by hamza221** [comment]
image
Does this look fine? there's no real visual distinction between the address and subject except the mail format

**Comment by ChristophWurst** [comment]
> Does this look fine? there's no real visual distinction between the address and subject except the mail format
> Also is the order fine ? Label->email->subject?

Looks good to me :)

File: src/components/ThreadEnvelope.vue [hunk 3]
```diff
@@ -60,6 +61,9 @@
 				@click.native.prevent="$emit('toggle-expand', $event)">
 				<div class="sender">
 					{{ envelope.from && envelope.from[0] ? envelope.from[0].label : '' }}
+					<p class="sender__email">
+						{{ envelope.from[0].email }}
```
Comment (ChristophWurst) [inline src/components/ThreadEnvelope.vue:65]: like on L63 we should save-guard a possibly missing sender. messages might not have a `from` address

**Review by ChristophWurst — APPROVED** [review]
:shipit:
